docker-compose.yml

```diff
@@ -25,3 +25,17 @@ services:
       AWS_S3_SECRET_ACCESS_KEY: ''
     ports:
       - "5000:5000"
+    container_name: gemini-server
+    links:
+      - postgres
+
+  postgres:
+    image: registry.centos.org/centos/postgresql-94-centos7:latest
+    network_mode: bridge
+    ports:
+     - "6432:5432"
```

Comment:
This means 'export port 5432 as 6432 to the outside word'.
fabric8-gemini-server service tries to access 5432 (`PGBOUNCER_SERVICE_PORT: "5432"`), so you need
```
ports:
     - "5432:5432"
```